pkg/sshd/etc/profile

```diff
@@ -0,0 +1,13 @@
+export CHARSET=UTF-8
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+export PAGER=less
+export PS1='(sshd ns):\w\$ '
+export PS1='\h:\w\$ '
```

Comment:
I would imagine there would be differences in the default binds, the command line, the set of installed packages, but similarities in the login arrangements and in things like PS1 munging.

The names would be `getty` and `sshd`, just like they are today.